### ISSUE TITLE
ClpSimplex::initialSolve fails and aborts

### ISSUE THREAD

**Issue by JockLawrie** [JuliaOpt/Cbc.jl#36]
Hi there,

I am solving several integer programs with somewhere between a few dozen and a few thousand variables. Almost every time the solution is found without any problem. However, for 1 model in particular I get the error below and the program aborts.

Also, if I add some noise to the vector of objective coefficients, the model solves without a hitch.

I ran basic summary statistics of the vector of objective coefficients for the failing model (mean, quantiles, sd, etc) and they are no different to those of the models that solve correctly.

Any ideas about what's wrong or what I could do to avoid this issue?
Is there a way to avoid the initialSolve function?

Cheers,
Jock

```
julia: ClpSolve.cpp:2442: int ClpSimplex::initialSolve(ClpSolve&): Assertion `!diffUpper || fabs(diffLower - diffUpper) < 1.0e-5' failed.

signal (6): Aborted
gsignal at /lib/x86_64-linux-gnu/libc.so.6 (unknown line)
abort at /lib/x86_64-linux-gnu/libc.so.6 (unknown line)
unknown function (ip: 0x7f11d0d2b266)
unknown function (ip: 0x7f11d0d2b312)
_ZN10ClpSimplex12initialSolveER8ClpSolve at /usr/lib/x86_64-linux-gnu/libClp.so.1 (unknown line)
_ZN21OsiClpSolverInterface12initialSolveEv at /usr/lib/x86_64-linux-gnu/libOsiClp.so.1 (unknown line)
_ZN8CbcModel12initialSolveEv at /usr/lib/x86_64-linux-gnu/libCbc.so.3 (unknown line)
_Z8CbcMain1iPPKcR8CbcModelPFiPS2_iER19CbcSolverUsefulData at /usr/lib/x86_64-linux-gnu/libCbcSolver.so.3 (unknown line)
Cbc_solve at /usr/lib/x86_64-linux-gnu/libCbcSolver.so.3 (unknown line)
optimize! at /home/jock/.julia/v0.4/Cbc/src/CbcSolverInterface.jl:134
jlcall_optimize!_22471 at  (unknown line)
jl_apply_generic at /home/jock/julia/bin/../lib/julia/libjulia.so (unknown line)
solve at /home/jock/.julia/v0.4/JuMP/src/solvers.jl:138
julia_solve_22357 at  (unknown line)
jl_apply_generic at /home/jock/julia/bin/../lib/julia/libjulia.so (unknown line)
solve_mip at /home/jock/repos/rch_surgery_remote.jl/mainsteps/compute_mip.jl:98
jlcall_solve_mip_22305 at  (unknown line)
jl_apply_generic at /home/jock/julia/bin/../lib/julia/libjulia.so (unknown line)
```

**Comment by JockLawrie** [comment]
I worked around the problem by installing the latest versions of `libCbcSolver` and `libClp`. [This post](http://list.coin-or.org/pipermail/clp/2015-October/001610.html) indicates the need to update to a version more recent than Oct 2015, whereas Cbc.jl's default Cbc version (2.9.1) predates that.

For reference I include a description of the steps I took (I am running Julia 0.4.5 on Linux Debian Jessie):

1) Move or remove the existing libCbcSolver and libClp (I just moved them in case anything went seriously wrong).
```
cd /usr/lib/x86_64-linux-gnu
sudo mv libCbcSolver.so.3 xxxlibCbcSolver.so.3
sudo mv libClp.so.1 xxxlibClp.so.1
```

2) `cd ~/.julia/v0.4/Cbc/deps`

3) Open `build.jl` and change line 36 (or thereabouts) from `cbcname = "Cbc-2.9.1"` to `cbcname = "Cbc-2.9.8"`.

4) In Julia run `Pkg.build("Cbc")`.
